installer/Install.ps1

```diff
@@ -86,6 +93,10 @@ If ((Test-Path -Path "MakePri.ps1") -Eq $true) {
 
 reg add "HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\AppModelUnlock" /t REG_DWORD /f /v "AllowDevelopmentWithoutDevLicense" /d "1"
 
+if ($PSHOME.contains("8wekyb3d8bbwe")) {
```

Comment:
Why?

Reply:
When using PowerShell which is installed via MS Store, `Get-WindowsOptionalFeature` and `Enable-WindowsOptionalFeature` will fail (See PowerShell/PowerShell#13866). Using the DISM module of Windows PowerShell will fix this.

Reply:
Some reports that install using Winget also facing the same issue, but can't confirm. https://github.com/PowerShell/PowerShell/issues/13866#issuecomment-1399276275

Reply:
In fact, users will encounter this as long as they install PowerShell with MSIX (Winget will download the MSIX and install it). So `$PSHOME.contains("8wekyb3d8bbwe")` can determine whether it is installed with MSIX